lib/python/milc.py

```diff
@@ -273,7 +285,7 @@ def description(self, value):
         self._description = self._arg_parser.description = self._arg_defaults.description = value
 
     def echo(self, text, *args, **kwargs):
-        """Print colorized text to stdout, as long as stdout is a tty.
+        """Print colorized text to stdout.
 
```

Comment:
At a passing glance, all the other comments have two lines beginning with `"""`, but not this one.

```suggestion
"""
```